<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>todoMainWindow</class>
 <widget class="QMainWindow" name="todoMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="completeButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>370</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Complete</string>
    </property>
   </widget>
   <widget class="QListView" name="todoView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>711</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="todoEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>711</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>711</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Todo</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
